### DISCUSSION TITLE
Has anyone a good idea to mock theme.other config for jest tests?

### DISCUSSION THREAD

**Discussion by Uniiq** [mantinedev/mantine#2056] [Q&A] [answered]
Hello,

i use custom configs in my theme. Has anyone an idea how i mock my `theme.other.test1.test2` values in my jest test?

Thanks a lot, Uniiq

**Comment by rtivital** [comment, marked as answer]
Hi @Uniiq, you can wrap your tests with MantineProvider that defines those values. See custom renderer setup documentation – https://testing-library.com/docs/react-testing-library/setup#custom-render

**Reply by Uniiq** [reply]
@rtivital thanks for the fast reply. Works like a charm.
